### DISCUSSION TITLE
通过Nginx Proxy Manager申请SSL证书时总是申请不成功

### DISCUSSION THREAD

**Discussion by duguyue8** [halo-dev/halo#4402] [Q&A]
!(1)[https://pic7.58cdn.com.cn/nowater/webim/big/n_v2fc25354902ec48f4add4f2f0632468c4.png]
!(2)[https://pic7.58cdn.com.cn/nowater/webim/big/n_v271f978e4b3b64aae8b24c8b1469578a4.png]

**Comment by duguyue8** [comment]
![1](https://pic7.58cdn.com.cn/nowater/webim/big/n_v2fc25354902ec48f4add4f2f0632468c4.png)
![2](https://pic7.58cdn.com.cn/nowater/webim/big/n_v271f978e4b3b64aae8b24c8b1469578a4.png)

**Reply by ruibaby** [reply]
看日志是因为未能正常注册 ACME 服务账号。
